### ISSUE TITLE
attribute OpenStreetMap properly (add it to the load screen as the first step)

### ISSUE THREAD

**Issue by matkoniecz** [mapsme/omim#11203]
Currently attribution of OSM data is effectively missing. It is present deeply hidden in settings, but it is effectively invisible.

To get it one needs to extended menu, settings menu, scroll down, "About MAPS.ME", scroll down, select "Copyright".

At the same time "MAPS.ME" credit text is always displayed in a bottom right corner of the map.

Proper attribution is present on the map, but only for about a second immediately after loading.

Interface is quite busy already, but at least on my phone load screen is present for about 6 seconds it is almost completely empty (first 2 seconds of a blank screen, later Maps.me logo, and "MAPS.ME" text). What about putting something like "powered by OpenStreetMap data" there as the first step?

I am using 9.1.8-Google, on Xiaomi Redmi 4, Android 6.0.1 MMB29M

**Comment by matkoniecz** [comment]
> Hi, we will add info to the loading screen in one of the upcoming releases. That is a good idea. Thanks!

What is the expected timeframe for doing this? Note that it is not "nice to have", it is a first step toward addressing fact that Maps.ME is committing large scale copyright violation by ignoring licese of OSM data (and without OSM data Maps.me is basically useless).

**Comment by thany** [comment]
It's funny that if you file an issue in a very "demanding" way, you get responses like "we have our own schedule" and "the app is free, stop being an arsehole", but at the same time, they improperly attribute the very data they rely on.

The contributors behind OSM do their work in their free time, giving you all this juicy data for absolutely free. And you have the *nerve* to not attribute them, esp since it is expressively in the OSM license. You should be deeply, deeply ashamed.

I'm writing this now, because this issue was opened a couple of months ago already, and there have been a couple of of releases since then.

**Comment by squatica** [comment]
Is the OSM attribution fixed in the latest version? It's been 7 months since @burivuh said it will be added.

**Comment by thany** [comment]
Otherwise we can start reporting the app to Google and Apple for violating license terms. Afaik, such reports are taken very seriously.

**Comment by najodleglejszy** [comment]
any updates about this issue? it's been a few months with no reply from the devs, and judging by a quick glance through the app it doesn't seem like anything has changed about crediting OSM.

**Comment by thany** [comment]
I think the MAPS.ME devs have already put this issue on an ignore list. Because it is so bleedingly simple to fix, and take so little effort. They just don't give a toss.

**Comment by moorchegue** [comment]
Just opened fresh installation of maps.me. OSM attribution is displayed for a couple of seconds in the bottom left corner and then it fades away.

**Comment by matkoniecz** [comment]
> OSM attribution is displayed for a couple of seconds in the bottom left corner and then it fades away.

Is MAPS.ME attribution still displayed continuously, or is it also displayed only for this few seconds?

AFAIK it is unchanged, this flash was already present - at least on some devices.

**Comment by moorchegue** [comment]
Yep, it's exactly like that. I guess it hasn't been changed then.

**Comment by ainola** [comment]
@burivuh it's coming up on a year since you expressed interest to start abiding by the terms of the license. What is the status?
